irispy/iris_tools.py

```diff
@@ -694,11 +918,12 @@ def _get_interpolated_effective_area(detector_type, obs_wavelength):
     # at which the data is recorded:
     eff_area_interp_base_unit = u.Angstrom
     tck = interpolate.splrep(response_wavelength.to(eff_area_interp_base_unit).value,
-                             eff_area.to(eff_area_interp_base_unit ** 2).value, s=0)
+                             eff_area.to(eff_area_interp_base_unit ** 2).value * iris_fit, s=0)
```

Comment:
This line should not be changed.  Instead this [line](https://github.com/sunpy/irispy/pull/108/files#diff-dee23d5475a6ed5e368938455c92fb2eR908) should be changed to
```python
iris_response = get_iris_response(**kwargs)
```
after the API has been changed in accordance with the above comment.

Reply:
Delete ```* iris_fit```